src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -2,210 +2,343 @@ project_path: /web/_project.yaml
 the `files` property of the event `target`.
 
-### Capture a single frame
+    <input type="file" accept="image/*" id="file-input">
+    <script>
+      const fileInput = document.getElementById('file-input');
+
+      fileInput.addEventListener('change', (e) => doSomethingWithFiles(e.target.files));
+    </script>
+
+The `files` property is a `FileList` object, which we'll talk more about later.
+
+You can also optionally add the `capture` attribute to the element, which indicates to the browser
+that you prefer getting an image from the camera.
+
+    <input type="file" accept="image/*" capture>
+    <input type="file" accept="image/*" capture="user">
+    <input type="file" accept="image/*" capture="environment">
+
+Adding the `capture` attribute without a value let's the browser decide which camera to use, while
+the `"user"` and `"environment"` values tell the browser to prefer the front and rear cameras,
+repectively.
 
-Getting access to the image file is simple.
+The `capture` attribute will work on Android and iOS, but be ignored on desktop. Be aware, however,
+that on Android this means that the user will no longer have the option of choosing an existing
+picture. The system camera app will be started directly, instead.
 
-    <input type="file" accept="image/*" capture="camera" id="camera">
-    <img id="frame">
+### Drag and drop
+If you are already adding in the ability to upload a file, there are a couple of easy ways that you
+can make the user experience a little richer.
+
+The first is to add a drop target to your page that allows the user to drag in a file from the
+desktop or another application.
+
+    <div id="target">You can drag an image file here</div>
     <script>
-      var camera = document.getElementById('camera');
-      var frame = document.getElementById('frame');
+      const target = document.getElementById('target');
 
-      camera.addEventListener('change', function(e) {
-        var file = e.target.files[0]; 
-        // Do something with the image file.
-        frame.src = URL.createObjectURL(file);
+      target.addEventListener('drop', (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        doSomethingWithFiles(e.dataTransfer.files);
+      });
+
+      target.addEventListener('dragover', (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        e.dataTransfer.dropEffect = 'copy';
       });
     </script>
 
-Once you have access to the file you can do anything you want with it. For 
-example, you can:
+Similar to the file input, you can get a `FileList` object from the `dataTransfer.files` property of
+the `drop` event;
 
-* Attach it directly to a `<canvas>` element so that you can manipulate it
-* Download it to the user's device
-* Upload it to a server by attaching to an `XMLHttpRequest` 
+The `dragover` event handler let's us signal to the user what will happen when they drop the file by
```

Comment:
I have done a search and replace 'we' with either 'I' or 'you', as appropriate, expect in the very last paragraph where 'We' obviously means 'Google'.

Similarly 'us' => 'you' and 'our' => your.